lux/vis/VisList.py

```diff
@@ -240,7 +240,7 @@ def sort(self, remove_invalid=True, descending=True):
         elif lux.config.sort == "descending":
             descending = True
         # sort in-place by “score” by default if available, otherwise user-specified field to sort by
-        self._collection.sort(key=lambda x: x.score, reverse=descending)
```

Comment:
Let's consider the case where we have a dropdown on the frontend for controlling the scoring, can we avoid regenerating the collection and simply resort based on the existing collection? This would require us to cache/store the generated collection before fetching the top K.